core/types/transaction_signing.go

```diff
@@ -232,11 +224,13 @@ func (fs FrontierSigner) Sender(tx *Transaction) (common.Address, error) {
 }
 
 func recoverPlain(sighash common.Hash, R, S, Vb *big.Int, homestead bool) (common.Address, error) {
+	V := byte(Vb.Uint64() - 27)
```

Comment:
Revert test code.

Reply:
Is this test code?? I thought it's the original code

Reply:
I didn't mean to delete this line. I mean revert these changes (the prints and the moved line)

Reply:
deleted the ops ...